src/Sylius/Bundle/CoreBundle/Test/Services/SecurityService.php

```diff
@@ -45,14 +49,23 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function logIn($email, $providerKey, Session $minkSession)
+    public function logIn($email, $providerKey = self::DEFAULT_PROVIDER_KEY, Session $minkSession)
```

Comment:
I'm curious what happens if we set `$providerKey` to something else, can you check if it is required?